service-templates/apigw-lambda-svc/v1/pipeline_infrastructure/example.terraform.tfvars

```diff
@@ -0,0 +1,12 @@
+# Copy+paste and rename to `terraform.tfvars` for working locally
+service_name = "terraform-test-service"
+
+codebuild_deployments = {
+  foo = {}
```

Comment:
in the code where this variable is used, I've added a `try(each.value.name, each.key)` which means - try to use the `name` attribute if its present (see next block where `bar` will be used via this route), if not present, then use the map's `key` which is `foo`